library/cartridge_backup_instance.py

```diff
@@ -97,13 +101,31 @@ def backup_stop(control_console):
     return None
 
 
-def call_backup(params):
-    start_only = params['start_only']
-    stop_only = params['stop_only']
+def custom_backup(params):
+    custom_backup_files = params.get('custom_backup_files')
+    assert custom_backup_files is not None, 'Parameter "custom_backup_files" is required'
+
+    # ARCHIVE
+    instance_id = params.get('instance_id')
+    assert instance_id is not None, 'Parameter "instance_id" is required'
+    backups_dir = params.get('backups_dir')
+    assert backups_dir is not None, 'Parameter "backups_dir" is required'
+    backup_archive_path = backup_pack(instance_id, backups_dir, custom_backup_files)
+
+    return helpers.ModuleRes(changed=True, fact={
+        'backup_archive_path': backup_archive_path,
+        'backup_files': custom_backup_files,
+    })
+
+
+def tnt_backup(params):
+    start_only = params.get('start_only', False)
+    stop_only = params.get('stop_only', False)
 
     assert not (start_only and stop_only), "impossible to use 'start_only' with 'stop_only'"
 
-    console_sock = params['console_sock']
+    console_sock = params.get('console_sock')
+    assert console_sock is not None, 'Parameter "console_sock" is required'
```

Comment:
What about smth like helpers.get_required_param(params, "name") instead of this copy-paste?